tests/DoctrineMongoODMModuleTest/Doctrine/CliTest.php

```diff
@@ -26,7 +34,7 @@
  * @link    http://www.doctrine-project.org/
  * @author  Adam Homsi <adam.homsi@gmail.com>
  */
-class CliTest extends PHPUnit_Framework_TestCase
+final class CliTest extends \PHPUnit_Framework_TestCase
```

Comment:
I can not merge this. We don't know how people are using CliTest, can you rollback to `class`?